src/middlewared/middlewared/plugins/enclosure_/jbof_enclosures.py

```diff
@@ -101,8 +101,23 @@ def map_es24n(model, rclient, uri):
     }
     mapped = dict()
     for disk in all_disks['Members']:
-        slot = int(disk['Id'])
-        if found := mounted_disks.get(disk['SerialNumber']):
+        if disk['State'] == 'Absent':
+            mapped[slot] = None
```

Comment:
`slot` is still unassigned here

Reply:
I think it's time for me to stop computer'ing today. 😄